public/assets/images/icons/icon-linkedin.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="3 3 27 27" fill="#FFFFFF" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M11.954,23h-2.95v-9.492h2.95V23z M10.449,12.151c-0.951,0-1.72-0.771-1.72-1.72c0-0.949,0.77-1.719,1.72-1.719c0.948,0,1.719,0.771,1.719,1.719C12.168,11.38,11.397,12.151,10.449,12.151z M23.004,23h-2.948v-4.616c0-1.101-0.02-2.517-1.533-2.517c-1.535,0-1.771,1.199-1.771,2.437V23h-2.948v-9.492h2.83v1.297h0.04c0.394-0.746,1.356-1.533,2.791-1.533c2.987,0,3.539,1.966,3.539,4.522V23z"/><path fill="none" stroke="#FFFFFF" stroke-miterlimit="10" stroke-width="2" d="M24.5,26h-17C6.672,26,6,25.328,6,24.5v-17C6,6.672,6.672,6,7.5,6h17C25.328,6,26,6.672,26,7.5v17C26,25.328,25.328,26,24.5,26z"/>
+</svg>
```

Comment:
Minor comment(this should not hold up merging haha), but the other icon file has a newline at the end of them so it would be nice to be consistent haha.